apps/explorer-e2e/src/integration/validator.cy.js

```diff
@@ -82,8 +83,98 @@ context('Validator page', { tags: '@smoke' }, function () {
                 // Therefore only checking the field name is present.
                 cy.contains('proposer_priority').should('be.visible');
               });
-          });
-      });
+            });
+        });
+    });
+
+    it('should be able to see relevant node information in vega data section', function () {
+      cy.get(vegaDataHeader)
+        .contains('Vega data')
+        .next()
+        .within(() => {
+          cy.get(jsonSection)
+            .invoke('text')
+            .convert_string_json_to_js_object()
+            .then((nodesInJson) => {
+              this.nodes.forEach((node, index) => {
+                const nodeInJson = nodesInJson.nodes[index];
```

Comment:
I think it'd be better to just loop through nodeInJson keys & values instead of writing each case explicitly.

Reply:
Unfortunately i can't do that, since some of the values are null, some are blank - Limitations of vegacapsule/config
As mentioned in the comments. Core will be working on unlocking more functionality, hopefully we can then loop through each key